libs/UGridNET/projs/UnitTestConsumer_VS-16-2019/Directory.Packages.props

```diff
@@ -0,0 +1,13 @@
+﻿<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Deltares.UGridNET" Version="[0.0.0-dev,)" />
```

Comment:
Eventually we want to not be dependend on the dev version but the signed (non dev version)

Reply:
Yes. The VS solutions are not used in production. for now they install an nupkg that is locally generated from the latest source. 
Ideally the signed nupk from the TC source should be used in a separate step.